micro-frontends/src/next-ui/Components/ProviderNotificationPatients/PatientsList.spec.jsx

```diff
@@ -2,30 +2,119 @@ import React from 'react';
 import { render, waitFor } from '@testing-library/react';
 import PatientsList from './PatientsList';
 
-const mockGetEmergencyDrugAcknowledgements = jest.fn().mockResolvedValue([]);
+const mockGetEmergencyDrugAcknowledgements = jest.fn();
+
+const mockData = [
```

Comment:
can we move mock datas to a separate file

Reply:
I've moved the data to seperate file